src/app/user-sign-up/login/login.component.scss

```diff
@@ -0,0 +1,76 @@
+@import '../../layout/_color.scss';
+
+/* Log in Form container */
+.form-container {
+    border-radius: 15px;
+    box-shadow: $offBlack 0px 5px 15px;
+    padding: 20px;
+    width: 400px;
+
+    /* Form group contains input, label tags, validation */
+    .form-group {
+        display: flex;
+        flex-direction: column;
+    
+        /* Email and password input */
+        input[type=password], [type=email] {
+            width: 100%;
+            padding: 12px 20px;
+            margin: 5px 0;
+            display: inline-block;
+            border: 1px solid $lightGrey1;
+            border-radius: 4px;
+            box-sizing: border-box;
+        }
+
+        // Validation for input tag styling
+        input.ng-invalid.ng-touched {
```

Comment:
Check how to write SCSS for this.
input.ng-invalid.ng-touched {
-> 
input{
   &.ng-invalid{
        &.ng-touched {
        }
  }
}